import-export-cli/cmd/importAPI.go

```diff
@@ -86,7 +86,7 @@ func init() {
 		"Preserve existing provider of API after importing")
 	ImportAPICmd.Flags().BoolVar(&importAPIUpdate, "update", false, "Update an "+
 		"existing API or create a new API")
-	ImportAPICmd.Flags().BoolVar(&importAPIRotateRevision, "rotate-revision", false, "Rotate the "+
+	ImportAPICmd.Flags().BoolVar(&importAPIRotateRevision, "rotate-revision", true, "Rotate the "+
```

Comment:
Do we need to change the rotate revision flag's default value in import API/API Product commands?
 - The first reason is, people might be already have written pipelines considering this behaviour, and this change might break those pipelines.
- The second reason is, these import commands do one import at a time. So the pipeline can capture the failure and check whether the import is failed due to a maximum number of revisions reached situation, if so then change the flag to true and import. Otherwise, if we keep true as the default value here, the oldest revision will be deleted automatically without the user's consent. (In VCS this is okay, because, since it is doing a bulk import, it is good to make the default value true. Also, since this behaviour is already broken in vcs we can make it true directly without considering existing pipelines. No harm will be done to vcs based pipelines)

WDYT? @1akshitha @uvindra

Reply:
Yes. Agree with Wasura's comment. Rotate revision will delete the oldest revision and this is a decision to be taken after evaluating each revision manually. This should not be done automatically. That's why we have introduced a flag to rotate if needed, not the other way around.

Reply:
Agreed with @wasuradananjith and @1akshitha. Will keep the default value as false in imports